Dockerfile

```diff
@@ -13,13 +13,19 @@ WORKDIR /go/src/github.com/flyteorg/flyteadmin
 # Pull dependencies
 RUN go mod download
 
+
 # COPY the rest of the source code
 COPY . /go/src/github.com/flyteorg/flyteadmin/
 
 # This 'linux_compile' target should compile binaries to the /artifacts directory
 # The main entrypoint should be compiled to /artifacts/flyteadmin
 RUN make linux_compile
 
+# Pull GRPC health probe binary for liveness and readiness checks
+RUN GRPC_HEALTH_PROBE_VERSION=v0.4.11 && \
```

Comment:
in terms of dockerfile layering, should this be before we run compile?

Reply:
Since we don't use this tool for flyteadmin compilation hence there is no issue with the ordering .

Any other reason you see where we would need this before

Reply:
I think we should move this before line 8 to avoid rerunning it with every change in go.mod or the code... otherwise looks good to me